test/e2e/network/netpol/kubemanager.go

```diff
@@ -34,13 +35,16 @@ import (
 
 // probeConnectivityArgs is set of arguments for a probeConnectivity
 type probeConnectivityArgs struct {
-	nsFrom         string
-	podFrom        string
-	containerFrom  string
-	addrTo         string
-	protocol       v1.Protocol
-	toPort         int
-	timeoutSeconds int
+	nsFrom               string
+	podFrom              string
+	containerFrom        string
+	addrTo               string
+	protocol             v1.Protocol
+	toPort               int
+	timeoutSeconds       int
+	maxAttempts          int
+	retryDelay           int
+	retryDelayMultiplier int
 }
```

Comment:
since this is for improving flakes: lets add a global counter for "totalProbeAttempts", and "totalRetries" and "average Retries per probe" ... that way EVEN IF this fixes our flakes, if some new CI system comes around later which has a high flake rate, we can look at the ratio of retries per probe , and say something like "overall, we see high numbers of probe retries being required, even on passing tests"... it will give us a heuristic we can use over time.